helm/cloud-provider-vsphere/charts/kube-vip-cloud-provider/values.yaml

```diff
@@ -0,0 +1,39 @@
+# Default values for kube-vip-cloud-provider.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicasCount: 1
+
+image:
+  repository: kubevip/kube-vip-cloud-provider
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.0.4"
+
+resources:
+  requests:
+    cpu: 50m
+    memory: 64Mi
+  limits:
+    cpu: 100m
+    memory: 128Mi
+
+tolerations:
+  - key: node-role.kubernetes.io/master
+    effect: NoSchedule
+  - key: node-role.kubernetes.io/control-plane
+    effect: NoSchedule
+
+affinity:
+  nodeAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 10
+        preference:
+          matchExpressions:
+            - key: node-role.kubernetes.io/control-plane
+              operator: Exists
+      - weight: 10
+        preference:
+          matchExpressions:
+            - key: node-role.kubernetes.io/master
```

Comment:
I would remove that one as it might fail in 1.24 and above

Reply:
Nope. It is `preferred` and the priority order is like
- control-plane
- master
- none